src/client/app/containers/view.mode.ts

```diff
@@ -63,11 +64,15 @@ export class ViewMode implements OnInit, OnDestroy {
     }
 
     get openInHostString() {
-        return this.strings.openInHost.replace('{0}', environment.current.host);
+        return this.strings.openInHost.replace('{0}', environment.current.host.toLowerCase());
     }
 
     get downloadAsHostFileString() {
-        return this.strings.downloadAsHostFile.replace('{0}', environment.current.host);
+        return this.strings.downloadAsHostFile.replace('{0}', environment.current.host.toLowerCase());
+    }
+
+    get tryItSupported() {
+        return window.localStorage.getItem('wacUrl') && environment.current.host.toUpperCase() === HostType.EXCEL;
```

Comment:
make `wacUrl` into constant at top of file, please.  and also prefix it with `playground`, or whatever we prefix our other variables with.